docs/HACKING.md

```diff
@@ -186,10 +186,10 @@ are commonly useful.
 
 ## Logging into a cluster live
 
-Login: `oc login https://api.ci.openshift.org/console/`
+Login: `oc login --server=https://api.ci.openshift.org`
```

Comment:
It will work to just drop the `/console` too i.e. `oc login https://api.ci.openshift.org`